src/VaultV2.sol

```diff
@@ -45,6 +45,7 @@ import {ISharesGate, IReceiveAssetsGate, ISendAssetsGate} from "./interfaces/IGa
 /// corresponding markets, and losses are deducted only when realized for these markets.
 /// @dev The caps are checked on allocate (where allocations can increase) for the ids returned by the adapter.
 /// @dev Relative caps are "soft" in the sense that they are only checked on allocate.
```

Comment:
```suggestion
/// @dev Relative caps are "soft" in the sense that they not checked on exit.
```

Reply:
no it's the fact that they are not checked on withdraw!

Reply:
@adhusson why are you proposing this?